frontend/device/remarkable/device.lua

```diff
@@ -17,17 +17,25 @@ local function getModel()
     end
     local model = f:read("*line")
     f:close()
-    return model == "reMarkable 2.0", model
+    return model == "reMarkable 2.0", model == "reMarkable Ferrari", model
```

Comment:
maybe `getModel()` could just return the model.

Then you can cache the model once `local model = getModel()`
and define the booleans as:

```lua
isRm2 = model =="reMarkable 2.0"
isRmPaperPro = model == "reMarkable Ferrrari"
```

??

Reply:
Done!